### ISSUE TITLE
#1338 - Filter button is not accessible with voice command using Dragon tool as no aria-label has been provided. 

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1649]
The attributes role and aria-label are missing. Looks like the fix for this issue has not been merged into the released package 4.5.0.

Added `role=button` and `aria-label` to match the text specified.

_Originally posted by @wobba in https://github.com/microsoft-search/pnp-modern-search/issues/1338#issuecomment-943440478_

**Comment by wobba** [comment]
Likely due to the changes @FranckyC did on filters.

**Comment by wobba** [comment]
Should be readded now in 4.5.2
